packages/@react-spectrum/tag/stories/TagGroup.stories.tsx

```diff
@@ -11,30 +11,54 @@
  */
 
 import {action} from '@storybook/addon-actions';
+import Audio from '@spectrum-icons/workflow/Audio';
+import {Icon} from '@react-spectrum/icon';
+import {Item} from '@react-stately/collections';
 import React, {useState} from 'react';
 import {storiesOf} from '@storybook/react';
-import {Tag, TagGroup} from '../src';
+import {TagGroup} from '../src';
+import {Text} from '@react-spectrum/text';
 
 storiesOf('TagGroup', module)
   .add(
     'default',
     () => render({})
-  ).add(
+  )
+  .add(
+    'items',
+    () => (
+      <TagGroup aria-label="tag group" items={[{key: '1', label: 'Cool Tag 1'}, {key: '2', label: 'Cool Tag 2'}]}>
+        {item =>
+          <Item key={item.key} aria-label={item.label}><Text>{item.label}</Text></Item>
+        }
+      </TagGroup>
+    )
+  )
+  .add('icons', () => (
+    <TagGroup aria-label="tag group" items={[{key: '1', label: 'Cool Tag 1'}, {key: '2', label: 'Cool Tag 2'}]}>
+      {item => (
+        <Item key={item.key} aria-label={item.label}>
+          <Icon>
+            <Audio />
+          </Icon>
+          <Text>{item.label}</Text>
+        </Item>
+      )}
+    </TagGroup>
+  ))
+  .add(
     'onRemove',
-    () => renderWithRemovableTags({
-      onRemove: action('onRemove')
-    })
-  ).add(
-    'is Disabled',
     () => render({
-      isDisabled: true
-    })
-  ).add(
-    'isReadOnly',
-    () => renderWithRemovableTags({
-      isReadOnly: true
+      isRemovable: true,
+      onRemove: action('onRemove')
     })
-  ).add(
+  )
+  .add(
+  'disabled',
+  () => render({
+    isDisabled: true
+  }))
+  .add(
     'with announcing',
     () => (
       <WithAnnouncing />
```

Comment:
This story will probably need to get refactored, think Jeff had it so ctrl + d created new tags? I'll have to check what exactly the previous behavior was but for now you could prob just tear out that extra stuff keydown stuff